src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -61,6 +62,18 @@ public void setScanNetworksListener(ScanNetworksListener listener) {
     scanNetworksListener = listener;
   }
 
+  /**
+   * Sets this DeviceController to use the given issuer for issuing operational certs.
+   * By default, the DeviceController uses an internal, OperationalCredentialsDelegate
+   * (see AndroidOperationalCredentialsIssuer)
+   * 
+   * @param issuer
+   */
+  public void setNOCChainIssuer(NOCChainIssuer issuer) {
+    setUseJavaCallbackForNOCRequest(deviceControllerPtr, issuer != null);
```

Comment:
Curious, rather than having two setters (one to set the value, one to set a flag indicating its presence) is it possible to use the presence or absence of this field to select automatically? That avoids the possibility of these two states being out of sync. i.e. if an issuer is not null, use it. Otherwise, use the existing behavior.

Reply:
I will consider this for a followup PR.